frontend/src/components/SingleRecipe.js

```diff
@@ -122,6 +137,62 @@ export default function SingleRecipe() {
     }
   };
 
+  const PDFGenerator = () => {
+    let doc = new jsPDF();
+    doc.setFontSize(22);
+    doc.text(20, 20, product.name);
+    doc.setFont('helvetica');
+    let bodyArr = [];
+    product.ingredients.map((i) =>
+      bodyArr.push({
+        ingredient: i.ingredient.name
+          .replace(/['Ő']/g, 'Ö')
```

Comment:
Szerintem az ékezetes betűkhöz ezt a részt kellene megnézni a doksiban: https://github.com/mrrio/jspdf#use-of-unicode-characters--utf-8

Reply:
Javítottuk!